admin/helpers/sce.js

```diff
@@ -0,0 +1,30 @@
+'use stict'
```

Comment:
Shouldn't the file be called `sce-presenter` to align with the naming convention used on other presenters?